### PR TITLE
Rename: Konnect Get Started section

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

What did you change and why?
- **Why:** Konnect is renaming certain features to better align with industry standards, so our docs need to be updated to reflect the new names
- **What:** Renamed features in the Get Started section of the docs. See the linked ticket for more info about which names are changing. 
 
Include any supporting resources, e.g. link to a Jira ticket, GH issue, FTI, Slack, Aha, etc.
DOCU-3370

### Testing instructions

https://deploy-preview-5961--kongdocs.netlify.app/konnect/getting-started/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/getting-started/configure-data-plane-node.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 ---
-title: Configure a Runtime
+title: Configure a Data Plane Node
```
Comment (cloudjumpercat) [inline app/konnect/getting-started/configure-data-plane-node.md:2]: ❗ For reviewers: Here and throughout this PR, if you could make sure I used the correct words in place of "runtime" that would be great. I sometimes had a hard time deciding if it should be control plane, data plane node, or data plane.

File: app/konnect/getting-started/configure-data-plane-node.md [hunk 2]
```diff
@@ -53,27 +53,26 @@ is part of the organization admin team by default.
     {{site.base_gateway}} instance and connects it to your
```
Comment (Guaris) [inline app/konnect/getting-started/configure-data-plane-node.md:53]:
```suggestion
    {{site.base_gateway}} and connects it to your
```

I can't add it as a suggestion but the "a" before KGW should be removed.

File: app/konnect/getting-started/configure-data-plane-node.md [hunk 3]
```diff
@@ -53,27 +53,26 @@ is part of the organization admin team by default.
...
+1. Once {{site.konnect_short_name}} detects your data plane node, 
+click **Done** to go back to the data plane nodes page.
 
-The runtime instances table will now include
+The data plane nodes table will now include
 a new entry for your instance and the tag in the **Sync Status** column should
 say **In Sync**.
 
-The default proxy URL for this runtime is `http://localhost:8000`. Take
+The default proxy URL for this control plane is `http://localhost:8000`. Take
```
Comment (Guaris) [inline app/konnect/getting-started/configure-data-plane-node.md:63]:
```suggestion
The default proxy URL for the data plane is `http://localhost:8000`. Take
```

File: app/konnect/getting-started/deploy-service.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: Proxy and Test a Service
 ---
 
-Create a {{site.konnect_short_name}} Gateaway service to proxy your APIs. In this guide, you will create, proxy, and test a gateway service using Runtime Manager in {{site.konnect_short_name}}. 
+Create a {{site.konnect_short_name}} Gateaway service to proxy your APIs. In this guide, you will create, proxy, and test a gateway service using Gateway Manager in {{site.konnect_short_name}}.
```
Comment (Guaris) [inline app/konnect/getting-started/deploy-service.md:5]:
```suggestion
Create a {{site.konnect_short_name}} Gateaway service to proxy your APIs. In this guide, you will create, proxy, and test a gateway service using the {{site.konnect_short_name}} Gateway Manager. 
```

File: app/konnect/getting-started/deploy-service.md [hunk 5]
```diff
@@ -52,8 +52,8 @@ In the {% konnect_icon runtimes %} [**Runtime Manager**](https://cloud.konghq.co
 ## Verify the implementation
 
 If you used the Docker script to create a container
-earlier in [Configure a Runtime](/konnect/getting-started/configure-runtime/),
-your runtime's default proxy URL is `localhost:8000`.
+earlier in [Configure a data plane node](/konnect/getting-started/configure-data-plane-node/),
+your control plane's default proxy URL is `localhost:8000`.
```
Comment (Guaris) [inline app/konnect/getting-started/deploy-service.md:56]:
```suggestion
your default proxy URL is `localhost:8000`.
```

File: app/konnect/getting-started/deploy-service.md [hunk 6]
```diff
@@ -12,11 +12,11 @@ specified in the API product deployment.
 
 If you're following the {{site.konnect_short_name}} quickstart guide,
 make sure you have
-[Configured a runtime](/konnect/getting-started/configure-runtime).
+[Configured a data plane node](/konnect/getting-started/configure-data-plane-node).
```
Comment (Guaris) [inline app/konnect/getting-started/deploy-service.md:15]:
```suggestion
[configured a data plane node](/konnect/getting-started/configure-data-plane-node).
```

File: app/konnect/getting-started/deploy-service.md [hunk 7]
```diff
@@ -52,8 +52,8 @@ In the {% konnect_icon runtimes %} [**Runtime Manager**](https://cloud.konghq.co
 ## Verify the implementation
 
 If you used the Docker script to create a container
```
Comment (Guaris) [inline app/konnect/getting-started/deploy-service.md:54]:
```suggestion
If you used the Docker script to connect a data plane
```

File: app/konnect/getting-started/index.md [hunk 8]
```diff
@@ -10,22 +10,22 @@ New to {{site.konnect_saas}}? Get started with the basics through the web app:
...
 
-1.  **Set up a runtime instance**:
+1.  **Set up a data plane node**:
 
-    Set up your first runtime instance, which gives you a data plane for proxying traffic.
+    Set up your first data plane node, which gives you a data plane for proxying traffic.
 
     Start with a data plane proxy so that when your service configuration is
     ready, your services are immediately connected, configured,
     and available for testing.
```
Comment (Guaris) [inline app/konnect/getting-started/index.md:19]:
```suggestion

```
This doesn't really make sense to me ( I know you didn't write it). But I think this should be rewritten to talk about automatically connecting your data planes using the script in the Gateway manager.

File: app/konnect/getting-started/index.md [hunk 9]
```diff
@@ -10,22 +10,22 @@ New to {{site.konnect_saas}}? Get started with the basics through the web app:
...
 
-    [Set up a runtime instance &gt;](/konnect/getting-started/configure-runtime/)
+    [Set up a data plane node &gt;](/konnect/getting-started/configure-data-plane-node/)
```
Comment (Guaris) [inline app/konnect/getting-started/index.md:21]:
```suggestion
    [Create a data plane node &gt;](/konnect/getting-started/configure-data-plane-node/)
```

I think configure  or connect could also work here, it depends on your perspective. The node is already created you run a script on it and it's configured to connect and work on Konnect. 
set up works better with something ephemeral like runtime instance.

File: app/konnect/getting-started/productize-service.md [hunk 10]
```diff
@@ -39,9 +39,9 @@ After creating the new version, you will see **Link with a Gateway Service** as
 
 1. Select **Link with a Gateway Service**. 
 
-    Choose a [runtime group](/konnect/runtime-manager/runtime-groups/) and [Gateway Service](/konnect/runtime-manager/configuration/#gateway-services) to
+    Choose a [control plane](/konnect/gateway-manager/control-plane-groups/) and [Gateway Service](/konnect/gateway-manager/configuration/#gateway-services) to
```
Comment (Guaris) [inline app/konnect/getting-started/productize-service.md:42]:
```suggestion
    Choose the [control plane](/konnect/gateway-manager/control-plane-groups/) and [Gateway Service](/konnect/gateway-manager/configuration/#gateway-services) to
```

File: app/konnect/getting-started/productize-service.md [hunk 11]
```diff
@@ -39,9 +39,9 @@ After creating the new version, you will see **Link with a Gateway Service** as
...
     deploy this API product version to. This lets you deploy to a specific group of
-    runtime instances in a specific environment.
+    data planes in a specific environment.
```
Comment (Guaris) [inline app/konnect/getting-started/productize-service.md:44]: It wont let me leave a suggestion here, but we should remove the use of specific twice here. Maybe "this lets you deploy your service across a data planes associated with the control plane"

**Review by Guaris — CHANGES_REQUESTED** [review]
Things I wasn't able to leave a comment on because of Github: 

Line 65 of `app/konnect/getting-started/deploy-service.md`

Runtime manager -> Gateway manager. 

Line 68 of `app/konnect/getting-started/deploy-service.md`

Runtime -> data plane


Let me know if I can help with anything.

**Review by acgoldsmith — APPROVED** [review]
LGTM

**Pull request merged**